translations/fr.pm

```diff
@@ -0,0 +1,172 @@
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+# REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+# AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+# INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+# LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+# OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+# PERFORMANCE OF THIS SOFTWARE.
+
+package Translation::fr;
+use strict;
+use parent 'Locale::Maketext';
+
+sub new () {
+  my $class = shift;
+  my $self = $class->SUPER::new();
+  $self->{encoding} = 'utf-8';
+  bless ($self, $class);
+  return $self;
+}
+
+our %Lexicon = (
+  _AUTO => 1,
+  _USAGE => '_USAGE',
+
+  'ERROR:' => 'ERREUR:',
+  'SOLUTION:' => 'SOLUTION:',
+
+  'Bad syntax or unknown instruction.' =>
+  'Mauvaise syntaxe ou instruction inconue.',
+
+  'Dear [_1],' =>
+  'Cher [_1]',
+
+  'Sincerely, the Minimalist' =>
+  "Sincèrement, Minimalist",
+
+  'You([_1]) are not subscribed to this list([_2]).' =>
+  "Vous([_1]) nêtes pas inscrit(e) à cette liste de diffusion([_2]).",
+
+  "Send a message to [_1] with a subject of 'help' (no quotes) for information about howto subscribe.",
+  "Envoyez un message à [_1] avec pour sujet « help » (sans guillement) pour obtenir de l'aide pour vous inscrire.",
+
+  'Your message follows:' =>
+  'Votre message est:',
+
+  'You([_1]) are not allowed to write to this list.' =>
+  "Vous([_1]) n'êtes pas auterisé à écrire à cette list.",
+
+  'Message size is larger than maximum allowed ([_1] bytes).' =>
+  'La taille du message excède le maximum autorisé [_1] octets).',
+
+  'Either send a smaller message or split your message into multiple smaller ones.' =>
+  'Envoyez en message plus court ou séparez le en plusieurs messages plus petits.',
+
+  'Your message header follows:' =>
+  'Les entêtes de votre message sont:',
+
+  'Internal error while processing your request; report sent to administrator.' =>
+  "Une erreur interne s'est produite pendant le traîtement de votre requête; un rapport est envoyé à l'administrateur.",
+
+  'Please note that subscription status for [_1] on [_2] did not change.' =>
+  "Veuillez noter que l'état de votre souscription pour  [_1] sur la liste [_2] n'a pas été modifié.",
+
+  'There is no authentication request with such code ([_1]) or the requst is invalid.' =>
+  "Il n'y a pas de requête d'authentification avec ce code ([_1]), ou la requête est invalide.",
+
+  'Resend your request to Minimalist.' =>
+  'Veuillez renvoyer votre requête à Minimalist.',
+
+  'You are not allowed to get subscriptions of other users.' =>
+  "Vous n'avez pas l'autorisation d'accéder aux inscriptions d'autres utilisateurs.",
+
+  'Current subscriptions of [_1]:' =>
```

Comment:
According to code it refers to an ``$email`` variable which appears to be an user, so my translation fits properly